src/shared/user-record-show.c

```diff
@@ -14,6 +14,8 @@
 const char *user_record_state_color(const char *state) {
         if (STR_IN_SET(state, "unfixated", "absent"))
                 return ansi_grey();
+        else if (streq(state, "degraded"))
```

Comment:
I'd prefer "dirty" here. "degraded" suggests something much more serious. "dirty" is also used everywhere in the code, so I think it'd be less confusing to use the same name here. Also, it shouldn't be red — it should be a recoverable condition, so yellow seems more appropriate.